src/backend/kvm/vm/builder.rs

```diff
@@ -38,7 +38,17 @@ pub trait Hook {
         Ok(())
     }
 
-    fn code_loaded(&mut self, _vm: &mut VmFd, _addr_space: &[u8]) -> Result<()> {
+    fn code_loaded(
+        &mut self,
+        _vm: &mut VmFd,
+        _addr_space: &[u8],
+        secret: *mut SevSecret,
+    ) -> Result<()> {
+        // Inject dummy secret
+        unsafe {
+            secret.write_bytes(0xEA, 1);
+        }
```

Comment:
KVM should return ENOSYS for `get_attestation()`, so I'm not sure the hypervisor should plant anything here in that case.

Reply:
ok